### ISSUE TITLE
update ckeditor to modern

### ISSUE THREAD

**Issue by tablatronix** [GetSimpleCMS/GetSimpleCMS#1137]
This has become essential due to the discontinuation of support of older browsers and needing new ckeditor for modern browsers.

**Comment by tablatronix** [comment]
Added ckeditor 4.5.7 with codesnippet and autogrow plugins
cache problems may arise

**Comment by cnb** [comment]
Not sure I like autogrow being enabled by default...

http://get-simple.info/forums/showthread.php?tid=8433

**Comment by tablatronix** [comment]
Yeah I was waiting for a complaint i mostly have it in for testing , i wrote my own version in 3.4 for snipets and wa t to see if this is a decent replacement. Auto setting to screen height is ideal

**Comment by tablatronix** [comment]
I realized there was no default maxheight after I released it.

I disabled the autogrow plugin.

I also added 2 toolbar plugins which make the autogrow plugin useful.

`fixed` fixed toolbar ( unfortunately it centers on page, so it could use some tweaking to match editor alignment )

`floating-tools` adds a floating toolbar which can be customized, and I included configurations for them.

They all need testing across browsers and mobile.
